src/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/NetStandardBridge.cs

```diff
@@ -0,0 +1,138 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+// This file makes NetStandard Reflection's "subclassing" surface area look as much like NetCore as possible so the rest of the code can be written without #if's.
+//
+
+namespace System.Reflection.TypeLoading
+{
+    //
+    // For code (usually shared with other repos) that have to interact with "Type" rather than "RoType", some handy extension methods that 
+    // "add" the NetCore reflection apis back to NetStandard.
+    //
+    internal static class NetCoreApiEmulators
+    {
+#if netstandard
```

Comment:
Consider using different files instead of if-defs?

Reply:
Done